common-types/src/main/protobuf/com/improving/app/common/domain/address.proto

```diff
@@ -3,6 +3,10 @@ syntax = "proto3";
 package com.improving.app.common.domain;
 import "scalapb/scalapb.proto";
 
+option (scalapb.options) = {
+  preserve_unknown_fields: false
+};
```

Comment:
This will prevent backwards compatibility in a future version. The unknown fields  are where it dumps extra stuff you've removed in the future.  Act with care an intentionality on this as it requires clairvoyance :)

Reply:
I will lean towards backwards compatibility and remove this. I realize that there's a high chance that things will break in the future because of this. Same for the comment below.

Reply:
OK